invisible_cities/cities/penthesilea_test.py

```diff
@@ -195,15 +197,15 @@ def test_penthesilea_produces_mcinfo(KrMC_pmaps_filename, KrMC_hdst, config_tmpd
 @mark.serial
 def test_penthesilea_true_hits_are_correct(KrMC_true_hits, config_tmpdir):
     penthesilea_output_path = os.path.join(config_tmpdir,'Kr_HDST_with_MC.h5')
-    penthesilea_evts        = load_mchits(penthesilea_output_path)
+    penthesilea_evts        = load_mchits_df(penthesilea_output_path)
     true_evts               = KrMC_true_hits.hdst
 
-    assert sorted(penthesilea_evts) == sorted(true_evts)
-    for evt_no, true_hits in true_evts.items():
-        penthesilea_hits = penthesilea_evts[evt_no]
-
-        assert len(penthesilea_hits) == len(true_hits)
-        (assert_MChit_equality(p_hit, t_hit) for p_hit, t_hit in zip(penthesilea_hits, true_hits))
+    assert np.all(penthesilea_evts.index.levels[0] == true_evts.index.levels[0])
```

Comment:
Since you are reading all as dataframes now isnt `assert_dataframes_equal(penthesilea_evts, KrMC_true_hits.hdst)` enough?
Also maybe use the `assert_dataframes_equal` from our testing utils?

Reply:
Is there any reason to use our own assert_dataframes_equal and not `pandas.testing. assert_frame_equal`?

Reply:
Maybe to be consistent with (mostly?) all other tests?  But also it might be easier to track in case something changes in future pandas/numpy version (ie, the testing used to be in pandas.utils.testing and now is pandas.testing...)
Also shouldn't assert close be more appropriate since those are floats?